test/test_simulation_results.jl

```diff
@@ -1044,6 +1045,24 @@ end
     first_result = first(thermal_results)
     last_result = last(thermal_results)
 
+    println(list_aux_variable_keys(results_ed))
+
+    loss_factors = read_aux_variable(results_ed, PowerFlowLossFactors, ACBus)
+
+    # here we check if the loss factors are stored in the results, the values are tested in PowerFlows.jl
+    @test loss_factors !== nothing
+
+    loss_factors = first(
```

Comment:
why are you getting the `loss_factors` two different ways?

Reply:
Just to make sure it works. Should I remove one of them?

Reply:
Yeah, I would say that the correspondence between `read_aux_variable` and `read_results_with_keys` is/ought to be tested elsewhere, so you should only need one of them